### DISCUSSION TITLE
Welcome to hirob-fe-website Discussions!

### DISCUSSION THREAD

**Discussion by husamettinarabaci** [Ubn-Jr/hirob-fe-website#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CemoAkcadogan** [comment]
Hello, I'm [@cemoakcadogan](https://github.com/CemoAkcadogan), and I'm the chairman of the [Hi-Rob-Fe-Website](https://github.com/orgs/Ubn-Jr/teams/hirob-fe) Team. My role within the group is to distribute tasks appropriately among everyone. Additionally, I am responsible for product marketing, including both physical and e-commerce aspects if necessary.

The main goal of [our team](https://github.com/orgs/Ubn-Jr/teams/hirob-fe) is to effectively promote the product by creating a modern website that aligns with current technology trends. We will focus on SEO efforts, integrate e-commerce functionalities, and, if required, facilitate product sales through physical markets. Each member of [our team](https://github.com/orgs/Ubn-Jr/teams/hirob-fe) will manage their respective domains, ensuring a seamless division of responsibilities. With Hi-Rob, children will not only have fun but also engage in educational activities.
